### DISCUSSION TITLE
(Global Env) and (Local Env) vs user created environments

### DISCUSSION THREAD

**Discussion by elgs** [thunderclient/thunder-client-support#1027] [Q&A] [answered]
What are the differences between the (Global Env)/(Local Env) and user created environments. I can set user created envs as active and they will be effective in the requests. However, it seems there is no way to set the (Local or Global Env) as active.

**Comment by rangav** [comment, marked as answer]
Hi @elgs 

Both local and global env are global environments which means they are available to all the collections.

Here is docs link for more info.
https://github.com/rangav/thunder-client-support#environments

**Comment by elgs** [comment]
Oh thanks for the explanation and the quick answer. Now I know how they work.
